cmd/inspector/main.go

```diff
@@ -39,6 +40,9 @@ var (
 	// CSVPath is the csv path where command output is written
 	CSVPath string
 
+	//DotPath is the output of command routing-graph
+	DotPath string
```

Comment:
Rather than saving into a file, output to stdout. It's nice to use and pipe to dot e.g. `inspector kad routing-graph | dot -Tsvg > routing-graph.svg`. Less flags is clearer.

Reply:
Hmm, I suspect `CSVPath` would've been nicer that way as well.